ui/blocks/Confirmation.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react'
+
+import { Button, Box, Text, Card } from '~ui/core'
+import { ArrowsRight } from '~ui/anicons'
+
+type ConfirmationProps = {
+  onSubmit: () => void
+}
+
+export const Confirmation: React.FC<ConfirmationProps> = ({ onSubmit }) => {
+  return (
+    <Card variant="form" mx={-4}>
+      <Text variant="h3" as="h3">
+        Checkin nur ohne Symptome
+      </Text>
+      <Box height={5} />
+      <Text>
+        <p>
+          Du darfst hier nur einchecken, wenn du keine Symptome von COVID19
+          aufweist.
+        </p>
+        <p>
+          Klicke auf Weiter um zu bestätigen, dass du keine Symptome aufweist.
+        </p>
```

Comment:
I guess we need `Sie` here as well? 🙈 Since it only is visible in the care env, we don't need both `du` and `sie`.
(Also the text was written by me and maybe it isn't the best 😅 )

Reply:
Done :+1: